lib/mix/lib/mix/task.ex

```diff
@@ -275,6 +277,14 @@ defmodule Mix.Task do
     end
   end
 
+  defp output_task_debug_info(task, args, proj) do
+    Mix.shell.info("** Running mix " <> task_to_string(task, args) <> project_to_string(proj))
+  end
+  defp project_to_string(nil), do: ""
```

Comment:
Very last comment, could you separate function definition with newlines. :bowtie: